Store/src/credits/credits.cs

```diff
@@ -6,20 +6,71 @@ namespace Store;
 
 public static class Credits
 {
+    // Get player credits safely
     public static int Get(CCSPlayerController player)

```

Comment:
It is more convenient for me to throw an error instead of pulling -1, so I used Single. If database works well, it will never be null.

Reply:
When not loaded, we should refuse to open the menu